core/src/test/java/org/ehcache/core/EhcacheBulkMethodsTest.java

```diff
@@ -68,14 +68,11 @@ public void testPutAll() throws Exception {
   @Test
   public void testGetAll() throws Exception {
     Store<Number, CharSequence> store = mock(Store.class);
-    when(store.bulkComputeIfAbsent((Set<? extends Number>)argThat(hasItems(1, 2, 3)), any(Function.class))).thenAnswer(new Answer<Object>() {
-      @Override
-      public Object answer(InvocationOnMock invocation) throws Throwable {
-        Function function = (Function)invocation.getArguments()[1];
-        function.apply(invocation.getArguments()[0]);
+    when(store.bulkComputeIfAbsent((Set<? extends Number>)argThat(hasItems(1, 2, 3)), any(Function.class))).thenAnswer(invocation -> {
+      Function function = (Function)invocation.getArguments()[1];
+      function.apply(invocation.getArguments()[0]);
 
-        return new HashMap(){{put(1, null); put(2, null); put(3, valueHolder("three")); }};
-      }
+      return new HashMap(){{put(1, null); put(2, null); put(3, valueHolder("three")); }};
```

Comment:
Has Mark Reynold said: Double curly braces are ugly.

Reply:
but collection factories are coming in 9 only ;-)

Reply:
That means 
```
Map map = new HashMap();
map.put(1, null);
map.put(2, null);
map.put(3, valueHolder("three"));
```

Reply:
My understanding is that this is actually creating a new subtype of HashMap with a non-static initializer that mutates the map during construction.  Bleagh.

Reply:
@chrisdennis Exactly. You get a branch new anonymous inner class each time